docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -83,13 +83,16 @@ Ceased Sidechain Withdrawal
 ===========================
 
 The funds of the ceased sidechain can be retained to the mainchain with Ceased Sidechain Withdrawal request. This request can be performed after 2 withdrawal epochs after sidechain ceasing.
-This request can be performed through a raw transaction with the following structure
+To perform this request on the sidechain side should be generated nullifier and Ceased Sidechain Withdrawal proof. Nullifier can be generated by api command nullifier(csw api gropu). Proof generation can be done with generateCswProof command.
```

Comment:
A line above: `This request can be performed after 2 withdrawal epochs after sidechain ceasing.` 
As far as I remember, the request can be performed just after ceasing.  We don't need to wait 2 epochs.